releasenotes/notes/2.2/parameterized_target_c-9c9a4ad2915c2348.yaml

```diff
@@ -4,8 +4,8 @@ features_c:
     The ``QkTarget`` type is now able to represent targets that support
     parameterized ``QkGate`` types that accept any parameter value. Previously,
     these gates could only be added to the target with a fixed angle value
-    supported by using ``qk_target_entry_new_fixed`` to create the ``QkTargetEntry``.
-    Now, the ``qk_target_entry_new`` function can be used with parameterized gates
+    supported by using :cpp:func:`qk_target_entry_new_fixed` to create the ``QkTargetEntry``.
+    Now, the :cpp:func:`qk_target_entry_new` function can be used with parameterized gates
     and when it is this indicates the gate in the target will support any value for
```

Comment:
```suggestion
    When it is used with parameterized gates, this function indicates that the gate in the target supports any value for
```

Reply:
Please double check this one to ensure that I didn't change the meaning.  I wasn't totally clear on what was being said.

Reply:
Comment on the below line:

"This creates a 5 qubit target that will accept an RZ gate on any qubit with any parameter value being supported" -> "This creates a five-qubit target that accepts an RZ gate on any qubit with any parameter value supported"